oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtIssuerAuthenticationManagerResolver.java

```diff
@@ -86,6 +86,16 @@ public JwtIssuerAuthenticationManagerResolver(Collection<String> trustedIssuers)
 				new TrustedIssuerJwtAuthenticationManagerResolver(
 						Collections.unmodifiableCollection(trustedIssuers)::contains));
 	}
+	
+	/**
+     * Construct a {@link JwtIssuerAuthenticationManagerResolver} using the provided
+     * parameters
+     * @param trustedIssuer a predicate to determine whether the issuer should be trusted or not
+     */
+    public JwtIssuerAuthenticationManagerResolver(Predicate<String> trustedIssuer) {
+        this.authenticationManager = new ResolvingAuthenticationManager(
+                new TrustedIssuerJwtAuthenticationManagerResolver(trustedIssuer));
+    }
 
```

Comment:
Are you able to add this same functionality for `JwtIssuerReactiveAuthenticationManagerResolver`?